lambdas/db-migration/src/migrations/20201014105045_create_executions_table.ts

```diff
@@ -24,6 +28,34 @@ export const up = async (knex: Knex): Promise<void> =>
       .references('cumulusId')
       .inTable('executions')
       .notNullable();
+    table
+      .text('url')
+      .comment('Execution page url on AWS console');
+    table
+      .enum('status', ['running', 'completed', 'failed', 'unknown'])
+      .comment('Execution status')
+      .notNullable();
+    table
+      .jsonb('tasks')
+      .comment('TODO');
+    table
+      .jsonb('error')
+      .comment('Error details in case of a failed execution');
+    table
+      .text('workflowName')
+      .comment('Name of the Cumulus workflow run in this execution');
+    table
+      .float('duration')
+      .comment('Execution duration');
+    table
+      .jsonb('originalPayload')
+      .comment('Original payload of this workflow');
+    table
+      .jsonb('finalPayload')
+      .comment('Final payload of this workflow');
+    table
+      .timestamp('timestamp')
```

Comment:
this seems redundant of `updatedAt`, but I'm not sure of the knock-on effects of eliminating it.

Reply:
Agreed, I think we don't want to add the complication of sorting out that technical debt right now